app/models/pension/charges/Charge.scala

```diff
@@ -17,9 +17,15 @@
 package models.pension.charges
 
 import play.api.libs.json.{Json, OFormat}
+import utils.Constants.zero
 import utils.EncryptedValue
 
-case class Charge(amount: BigDecimal, foreignTaxPaid: BigDecimal)
+case class Charge(amount: BigDecimal, foreignTaxPaid: BigDecimal) {
+
+  def isBlankSubmission: Boolean =
+    if (amount == zero && foreignTaxPaid == zero) true
```

Comment:
you can just return `(amount == zero && foreignTaxPaid == zero)` if not needed

Reply:
Oh yes. Didn't spot that haha.